Routes.js

```diff
@@ -37,28 +36,46 @@ export default createAppContainer(createStackNavigator(
 		UsersSelect: { screen: UsersSelect },
 		CommentsList: { screen: CommentsList },
 		CommentsEdit: { screen: CommentsEdit },
-		SettingsList: { screen: SettingsList },
+		SettingsList: { screen: SettingsList }
 	},
 	{
+		initialRouteName: 'SitesList',
 		navigationOptions: {
 			headerStyle: {
-				backgroundColor: 'white',
+				backgroundColor: "white",
 				borderBottomWidth: 0,
-				shadowColor: 'transparent',
+				shadowColor: "transparent",
 				shadowRadius: 0,
 				shadowOffset: {
-					height: 0,
-				},
-			},
+					height: 0
+				}
+			}
 		},
 		cardStyle: {
-			backgroundColor: 'white',
+			backgroundColor: "white",
 			borderTopWidth: 0,
 			shadowRadius: 0,
 			shadowOffset: {
-				height: 0,
+				height: 0
 			},
-			shadowColor: 'transparent',
+			shadowColor: "transparent"
 		},
 	}
-));
+);
+
+export default createAppContainer(
+	createStackNavigator(
+		{
+			Main: {
+				screen: mainStack
+			},
+			SitesAdd: {
```

Comment:
Might be better to call this something different so it's clearer which is which.